src/aae_exchange.erl

```diff
@@ -801,7 +809,7 @@ compare_branches(BlueBranches, PinkBranches) ->
         end,
     lists:foldl(FoldFun, [], lists:seq(1, length(BlueBranches))).
 
--spec compare_clocks(list(tuple()), list(tuple())) -> list(tuple()).
+-spec compare_clocks(list(tuple()), list(tuple())) -> list(repair_input()).
 %% @doc
 %% Find the differences between the lists - and return a list of
 %% {B, K, blue-side VC, pink-side VC}
```

Comment:
Here in the comment we return a 4-tuple, but repair-input is a tuple of 2-tuples.

Reply:
the comment is wrong, I will correct